hm:

```
/usr/bin/ld: /pytorch/executorch/build-x86/lib/libextension_module.so: undefined reference to `vtable for executorch::extension::MmapDataLoader'
/usr/bin/ld: /pytorch/executorch/build-x86/lib/libextension_module.so: undefined reference to `executorch::extension::MmapDataLoader::~MmapDataLoader()'
/usr/bin/ld: /pytorch/executorch/build-x86/lib/libextension_module.so: undefined reference to `executorch::extension::MmapDataLoader::from(char const*, executorch::extension::MmapDataLoader::MlockConfig)'
```

--- a/extension/data_loader/targets.bzl
+++ b/extension/data_loader/targets.bzl
@@ -71,11 +71,11 @@ def define_common_targets():
         name = "mmap_data_loader",
         srcs = [
             "mmap_data_loader.cpp"
-        ] + ["mman_windows.cpp"] if host_info().os.is_windows else [],
-        exported_headers = [
+        ] + (["mman_windows.cpp"] if host_info().os.is_windows else []),
+        headers = [
             "mman.h",
-            "mmap_data_loader.h",
-        ] + ["mman_windows.h"] if host_info().os.is_windows else [],
+        ] + (["mman_windows.h"] if host_info().os.is_windows else []),
+        exported_headers = ["mmap_data_loader.h"],
         visibility = [
             "//executorch/test/...",
             "//executorch/extension/pybindings/...",

